### DISCUSSION TITLE
Create an account and get back a seed phrase only using backend

### DISCUSSION THREAD

**Discussion by IMFefer** [celo-org/react-celo#128] [Q&A]
Hi, I was trying to create an account only using back end and getting back a seed phrase, so the user can directly acces the wallet without going throught all the process of creating one, everything done on telegram with a bot.

**Comment by arthurgousset** [comment]
Hey @IMFefer 👋 If I understand you correctly (from our conversation earlier), you would like help generating seed phrases for Celo users. 

For reference: 
- @critesjosh (previously) kindly linked you to a [JS implementation of BIP39 available on npm](https://www.npmjs.com/package/bip39)
- I also came across a "[ReactNative-ready](https://github.com/celo-org/react-native-bip39) fork of the original Javascript implementation of Bitcoin BIP39 [above 👆]"

It seems that you were having difficult using the JS library with your telegram bot setup. Perhaps you can describe the specific problem you are facing more. 

N.B: I'm not certain the the [use-contractkit repo](https://github.com/celo-org/use-contractkit) discussion is the best place for this. 

Perhaps @aaronmgdr can help? :)

**Comment by aaronmgdr** [comment]
use-contractkit is meant exclusively for the front-end. For backend You want [@celo/contractKit](https://www.npmjs.com/package/@celo/contractkit)

**Reply by IMFefer** [reply]
Do you have the github repo for that?

**Reply by critesjosh** [reply]
https://github.com/celo-org/celo-monorepo/tree/master/packages/sdk
